nntrainer/tensor/cache_pool.cpp

```diff
@@ -56,7 +56,7 @@ convertTensorLifespanToCachePolicy(const TensorLifespan lifespan) {
     policy = CachePolicy::TEMPORAL;
     break;
   case TensorLifespan::CALC_GRAD_DERIV_LIFESPAN:
-    policy = CachePolicy::TEMPORAL;
+    policy = CachePolicy::ITERATION_CONSIST;
```

Comment:
calc_grad_deriv is part of the backwarding only. It seems not the case of Iteration_consist. 
Do you define iteration differently? Seems like we need more fine-grained matching though.

Reply:
`ITERATION_CONSIST` in swap cache means its value keeps its consistency during iteration from the start to the end of its life. So, in this case (CALC_GRAD_DERIV_LIFESPAN), it keeps its value from GRAD to DERIV. and it abandons its value after DERIV.
 Sorry for the little difference meaning of `ITERATION`. I'll add comments for it.